### PR TITLE
New diagram for the introduction page + small tweaks

### PR DESCRIPTION
**Pull request by rkoron007**

### CODE REVIEW

File: docs/source/index.mdx [hunk 1]
```diff
@@ -4,13 +4,14 @@ title: Introduction to Apollo Server
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-![diagram](./images/index-diagram.svg)
+![diagram](./images/frontend_backend_diagram.svg)
```
Comment (StephenBarlow) [inline docs/source/index.mdx:7]:
```suggestion
<img src="./images/frontend_backend_diagram.svg" width="500" alt="Diagram showing Apollo Server bridging frontend and backend" />
```

Comment (StephenBarlow) [reply docs/source/index.mdx:7]: (The diagram is a little too big by default, and we can more easily shrink it a bit with raw HTML like this)

**Review by StephenBarlow — APPROVED** [review]
Approved with a tweak! Please double-check that my HTML change makes the image show up correctly 😄

File: docs/source/index.mdx (hunk 1 above)
Comment (rkoron007) [reply docs/source/index.mdx]: Looks great! Thank you so much!

File: docs/source/index.mdx [hunk 2]
```diff
@@ -4,13 +4,14 @@ title: Introduction to Apollo Server
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-![diagram](./images/index-diagram.svg)
+<img src="./images/frontend_backend_diagram.svg" width="500" alt="Diagram showing Apollo Server bridging frontend and backend" />
 
 #### You can use Apollo Server as:
 
+* A gateway for a [federated supergraph](/federation/)
+  * The GraphQL server for a [subgraph](/federation/subgraphs) in a federated supergraph
```
Comment (glasser) [inline docs/source/index.mdx:12]: @rkoron007  Was this intended to be more deeply nested?

File: docs/source/index.mdx [hunk 3]
```diff
@@ -4,13 +4,14 @@ title: Introduction to Apollo Server
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-![diagram](./images/index-diagram.svg)
```
Comment (glasser) [inline docs/source/index.mdx:7]: Should we delete the old diagram?

Comment (rkoron007) [reply docs/source/index.mdx:7]: Oops, yes we should! I'll make a new PR —thanks for catching that!

File: docs/source/index.mdx (hunk 2 above)
Comment (rkoron007) [reply docs/source/index.mdx:12]: This was meant to be nested, but on second look I think I would like these all to be on the same level. I'll remove that indent. :)

**Pull request merged**